docs/source/index.md

```diff
@@ -1,8 +1,4 @@
 ```{include} ../../README.md
----
```

Comment:
No longer needed since all the images are URLs to GitHub raw files (otherwise the images don't show on PyPI)